pkg/libvirttools/virtualization.go

```diff
@@ -54,6 +55,9 @@ const (
 	diskLetterStr               = "bcdefghijklmnopqrstu"
 
 	podNameString = "@podname@"
+
+	vmLogBaseDir  = "/var/log/vms"
```

Comment:
Should this be configurable somehow to work with [this setting ](https://github.com/Mirantis/virtlet/pull/281/files#diff-cbd52dd8eefa7bebcdd2b305c4bed4afR98)?